include/saber/geometry/size.hpp

```diff
@@ -40,6 +40,10 @@ class Size
     Size& operator*=(const Size& inSize);
     Size& operator/=(const Size& inSize);
 
+     // Logical operators
+    bool operator==(const Size& inSize);
+    bool operator!=(const Size& inSize);
```

Comment:
Same issue here. "free functions" rather than "instance methods"... and never "virtual"